packages/reference/src/common/EntityStore.tsx

```diff
@@ -204,7 +259,17 @@ async function fetchExternalResource({
             resourceTypeId: resourceType,
             query: { 'sys.urn[in]': urn },
           })
-          .then(({ items }) => items[0] ?? null),
+          .then(({ items }) => {
+            return items[0] ?? null;
+          })
+          .catch((e) => {
+            const parsedError = JSON.parse(e.message);
+            if (isFunctionInvocationErrorResponse(parsedError)) {
+              resourceFetchError = parsedError;
+            }
+
+            return null;
```

Comment:
Previously the error was not handled and therefore thrown to the parent, now we would hide maybe some errors, is this on purpose?

Reply:
We can't rethrow the original error here because we need the `resourceTypes` call to finish in order to get the correct appDefinition id and organization id that is required to generate the function log link in the FunctionInvocationErrorCard component.

I considered making these two calls to the CMA client sequential, but that would cause a degradation in performance.

I could store the original error in the `resourceFetchError` variable declared in the outer scope and just rethrow it if it's not a `FunctionInvocationResponseError` after the CMA client calls are complete.

I'm not sure if that's much better.

Reply:
I changed this to rethrow the original error if it's not related to function invocation.